R/write_vc.R

```diff
@@ -106,12 +106,16 @@ write_vc.character <- function(
   meta_data[["..generic"]][["git2rdata"]] <- as.character(
     packageVersion("git2rdata")
   )
-  meta_data[["..generic"]][["data_hash"]] <- hashfile(file["raw_file"])
+  meta_data[["..generic"]][["data_hash"]] <- datahash(raw_data)
```

Comment:
No, that's the point I mentioned above: keyboard imported characters (and unicode characters written in code) need no conversion in Windows.  If they would be converted, they would get a different hash code (and 3 tests would fail in AppVeyor).  (Characters from text files, on the other hand, need to be converted.)